Source/WTF/wtf/threads/Signals.h

```diff
@@ -89,6 +100,7 @@ inline Signal fromSystemSignal(int signal)
     default: return Signal::Unknown;
     }
 }
+#endif // OS(UNIX)
```

Comment:
This could be a follow up but I would move these into the `Signals.cpp` file. My grepping showed that neither function was used outside of `Signals.cpp`.

Reply:
That sounds good.